example/test/kotlin/example/ExpectationTest.kt

```diff
@@ -0,0 +1,27 @@
+package example
+
+import com.github.codefabrikgmbh.scenarios.given
+import example.setup.ExpectationException
+import example.setup.`expection scenario`
+import org.junit.Test
+
+
+class ExpectationTest {
+    @Test
+    fun `run without expectation`() {
```

Comment:
Please use the general test naming convention in this repository `<doing something> should <expectation>`. In this case `running a scenario should work`. 
The test here actually is `setting a variable should work` but this is not what we want to test. A test without a test body would be equally good.